### DISCUSSION TITLE
Excel-Like ou DataSheet, have any plugin for?

### DISCUSSION THREAD

**Discussion by alexworldbr** [filamentphp/filament#6883] [General]
Hi Guys,

Have any plugin ou suggested code/approach to edit data in excel-like format for Filament?

Similar to PowerGrid, with inline edit, copy/paste, selectable cells, click and arrow, orderby colunm etc, but for Filament.

Have a amazing reactjs snipet like https://github.com/farseerdev/sheet-happens, would possible use in Filament?

Thanks.

**Comment by pboivin** [comment]
Hi @APMU, this is not quite what you need but have a look at this plugin : 
https://filamentphp.com/plugins/awcodes-table-repeater

I don't know any other plugin to edit tabular data... Maybe you can create a custom field using the React package you found : 
https://filamentphp.com/docs/2.x/forms/fields#building-custom-fields

**Comment by shahzadwaris** [comment]
@APMU did you find any solution for this?
